### PR TITLE
Setting up DI and dependencies

### PR DESCRIPTION
**Pull request by Luitame**
- Set up the services
- Moved all Symfony dependencies to 3.4 version

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,8 +20,10 @@
     },
     "require": {
         "php": "^7.1",
-        "symfony/config": "^3.0",
-        "symfony/dependency-injection": "^3.0",
-        "symfony/http-kernel": "^3.0"
+        "symfony/config": "^3.4",
```
Comment (ianrodrigues) [inline composer.json]: I think we can support both 3.4 and 4+
```suggestion
        "symfony/config": "^3.4||^4.0",
```

File: src/Resources/config/services.yml [hunk 2]
```diff
@@ -1,4 +1,10 @@
 services:
-#    salesforce.example:
-#        class: Oowlish\Example
-#        arguments: ["@service_id", "plain_value", %parameter%]
+    GuzzleHttp\Client:
```
Comment (ianrodrigues) [inline src/Resources/config/services.yml]: I'm not sure about this indentation. It seems weird to me.

File: composer.json (hunk 1 above)
Comment (Luitame) [reply composer.json]: Are you sure? I don't have sure yet if we are ensuring the compatibility with Symfony 4 and its variants.

File: src/Resources/config/services.yml (hunk 2 above)
Comment (Luitame) [reply src/Resources/config/services.yml]: Yes, for sure. I think that I'm wrong. Is it possible to put that validation on StyleCI to yaml files?

File: composer.json (hunk 1 above)
Comment (ianrodrigues) [reply composer.json]: Yes, it is.

File: src/Resources/config/services.yml (hunk 2 above)
Comment (ianrodrigues) [reply src/Resources/config/services.yml]: I don't think so.

File: composer.json (hunk 1 above)
Comment (Luitame) [reply composer.json]: Done! :+1:

File: src/Resources/config/services.yml (hunk 2 above)
Comment (Luitame) [reply src/Resources/config/services.yml]: I already put this settings on the `.editorconfig` file. Thx!

**Pull request merged**